include/seastar/core/reactor_config.hh

```diff
@@ -32,6 +32,7 @@ struct reactor_config {
     bool handle_sigint = true;
     bool auto_handle_sigint_sigterm = true;
     unsigned max_networking_aio_io_control_blocks = 10000;
+    bool kernel_page_cache = false;
```

Comment:
It's good to move these configurations to reactor_config, this will allow applications to drop the use of Seastar provided boost::program_options eventually, or use them as just helpers.